.changeset/icy-webs-help.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+---
+---
+"section": legacy
+---
+
+Removed uses of ensureNoDataModelChanges.
```

Comment:
Might be more accurate to say something like "Remove `IFluidParentContext.ensureNoDataModelChanges and its implementations". We should also be sure to list all of the affected (externally facing) types in this doc for completeness.